src/components/atoms/SfAlert/SfAlert.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../../css/variables';
+
+$sf-alert-padding: 0.625rem !default;
+
+$sf-alert__text-font-size: 0.875rem !default;
```

Comment:
sf-alert__text-font-size => `sf-alert-font-size` + correct everything else according to docs